enumfields/enums.py

```diff
@@ -114,6 +115,10 @@ def class_to_str(cls):
 
 
 class ChoicesEnum(enum.Enum, metaclass=ChoiceEnumMeta):
+    value: Any
+    label: str
+    initial: bool
+    next: list[str] | None
```

Comment:
This is not the nicest solution but for our use case is sufficient enough.

Mypy has a problem inferring this information from the dynamic construction of the enum class. These annotations will help MyPy with the majority of our use cases. It won't work for cases where we use non-standard keys (provided through `kwargs` of `Choice`) - in these cases, we need to annotate the special keys on our side.